docs/reference/quadratic.rst

```diff
@@ -0,0 +1,217 @@
+===============================
+Quadratic Models: Unconstrained
+===============================
+
+.. _qm:
```

Comment:
For consistency: we usually put this tag at the top of the file

Reply:
I put this one and `_bqm` at the relevant section. Seems more useful that way to me?

Reply:
We use these to reference the page; for example, 

![image](https://user-images.githubusercontent.com/34041130/128387388-fea0d4a7-a93d-4539-949f-6c04725590ca.png)

And the text displayed comes from the subsequent header. We'd want that to be "Quadratic Models: Unconstrained", no?

Reply:
I think we'd still want it to say `Binary Quadratic Models`. I think I am going to leave it for now and you can change it back when you work on it :smile: